<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTreeWidget" name="bucketList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>771</width>
      <height>192</height>
     </rect>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="autoExpandDelay">
     <number>-1</number>
    </property>
    <property name="indentation">
     <number>20</number>
    </property>
    <property name="headerHidden">
     <bool>false</bool>
    </property>
    <property name="expandsOnDoubleClick">
     <bool>true</bool>
    </property>
    <property name="columnCount">
     <number>4</number>
    </property>
    <attribute name="headerDefaultSectionSize">
     <number>200</number>
    </attribute>
    <column>
     <property name="text">
      <string>S3 Bucket Items</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>SRT Progress</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>MD5 Checksum</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Filesize</string>
     </property>
    </column>
   </widget>
   <widget class="QTreeView" name="bucketView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>771</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="refreshBucketList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Refresh Buckets</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnTranscribe">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>440</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Transcribe</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
